src/compiler/GraphCompiler.ts

```diff
@@ -106,6 +107,19 @@ class GraphCompiler {
     });
   }
 
+  private getPrimitiveType(type: string = "string"): string {
+    switch(type) {
+      case "number":
+        return "Float";
+      case "boolean":
+        return "Boolean";
+      case "string":
+      default:
```

Comment:
Return string value in default case

Reply:
I Added a parameter like a standard return